Cabal/Distribution/Simple/GHC.hs

```diff
@@ -657,14 +657,14 @@ buildOrReplLib forRepl verbosity numJobs pkg_descr lbi lib clbi = do
     info verbosity "Building C++ Sources..."
     sequence_
       [ do let baseCxxOpts    = Internal.componentCxxGhcOptions verbosity implInfo
-                               lbi libBi clbi libTargetDir filename
+                                lbi libBi clbi libTargetDir filename
```

Comment:
Still a spurious whitespace change here

Reply:
Don't we want the arguments to be vertically aligned the the function call above?

Reply:
Yeah, sure. So long as it's deliberate and not just random reformatting noise.